dosomething-dev/main.tf

```diff
@@ -1,8 +1,56 @@
+variable "fastly_api_key" {}
+variable "heroku_email" {}
+variable "heroku_api_key" {}
+variable "papertrail_destination" {}
+variable "papertrail_destination_fastly" {}
 variable "northstar_pipeline" {}
 variable "phoenix_pipeline" {}
 variable "rogue_pipeline" {}
-variable "papertrail_destination" {}
-variable "papertrail_destination_fastly" {}
+
+terraform {
+  backend "remote" {
+    organization = "dosomething"
+
+    workspaces {
+      prefix = "dosomething-"
```

Comment:
This could also easily be `name = "dosomething-dev"`, but this eases us into a future where we have a single `dosomething` module that gets applied to `-dev`, `-qa`, and `-prod` workspaces with different variables.

Reply:
I'll tackle that in a follow-up ticket since this already got so much heftier than I'd imagined it would.